Jenkinsfile

```diff
@@ -6,9 +6,14 @@
  */
 @Library(['private-pipeline-library', 'jenkins-shared']) _
 
+def seleniumDockerImage = 'selenium/standalone-chrome'
+def seleniumDockerVersion = '3.141.59-20200409'
+
 dockerizedBuildPipeline(
   prepare: {
     githubStatusUpdate('pending')
+
+    sh "docker run --name selenium-chrome -d -p 4444:4444 $seleniumDockerImage:$seleniumDockerVersion"
```

Comment:
Minor, but I generally prefer the curly brace form of groovy string interpolation.  Especially when using the `sh` call.  

https://docs.sonatype.com/display/CDI/Jenkins+Pipeline+Shell+Script+Gotcha%27s#JenkinsPipelineShellScriptGotcha's-recRecommendations